tests/components/google_assistant/test_google_assistant.py

```diff
@@ -359,19 +319,6 @@ def test_execute_request(hass_fixture, assistant_client):
                             "brightness": 70
                         }
                     }]
-                }, {
-                    "devices": [{
-                        "id": "light.kitchen_lights",
-                    }],
-                    "execution": [{
-                        "command": "action.devices.commands.ColorAbsolute",
-                        "params": {
-                            "color": {
-                                "spectrumRGB": 16711680,
```

Comment:
According to Google, a light is either in Color spectrum or color temperature mode. With spectrum being able to do all colors, temperature mode is just for whites.

https://developers.google.com/actions/smarthome/traits/colortemperature#device-states
> Since a given light is in spectrum OR temperature mode, this object includes the current color settings in the relevant mode.